cmd/cape/cmd/deploy.go

```diff
@@ -63,6 +64,8 @@ func init() {
 	deployCmd.PersistentFlags().StringP("name", "n", "", "a name to give this function (default is the directory name)")
 	deployCmd.PersistentFlags().StringSliceP("pcr", "p", []string{""}, "pass multiple PCRs to validate against")
 	deployCmd.PersistentFlags().BoolP("public", "", false, "make the function public (anyone can run it)")
+
+	registerTemplate(deployCmd.Name(), deployTmpl)
```

Comment:
does `tmpl['deploy']` get registered before line 200 in `root.go`?

https://github.com/capeprivacy/cli/pull/194/files#diff-73ed9e69789bea01d1969c0ba24319d1bbb901e6605b3eeb0b76156c5097b08eR200

Reply:
That's called [here](https://github.com/capeprivacy/cli/pull/194/files#diff-73ed9e69789bea01d1969c0ba24319d1bbb901e6605b3eeb0b76156c5097b08eR40) in the `PersistentPreRunE` method, which won't be called until the command is executed. This is in an init func which will be called before execution of the main program starts. https://go.dev/doc/effective_go#init

Reply:
Oops I missed this. Thanks!